packages/arb-ts/src/lib/bridge_helpers.ts

```diff
@@ -141,6 +167,22 @@ export class BridgeHelper {
     )
   }
 
+  static calculateRetryableAutoReedemTxnHash = async (
```

Comment:
I don't particularly like this name, how about `calculateUserTxFromRetryable`

Reply:
@edfelten